### PR TITLE
fixup: Fix problem that scrollToPosition is not working correctly

### PR DESCRIPTION
**Pull request by HunseopJeong**
[Problem] Scroll is not working correctly when using the scrollToIndex
[Solution] Restore the default position of scroller when destroying the
snaplistview

Signed-off-by: Hunseop Jeong <hs85.jeong@samsung.com>

### CODE REVIEW

File: src/js/profile/wearable/widget/wearable/SnapListview.js [hunk 1]
```diff
@@ -683,6 +683,7 @@
 				self._selectedIndex = null;
 				self._currentIndex = null;
 
+				scrolling.scrollToIndex(0, 0);
```
Comment (TomaszLukawskiSam) [inline src/js/profile/wearable/widget/wearable/SnapListview.js]: I'm not sure this change. This patch causes unwanted effects during the transition animation between pages. User can see items from top of page during page transition.

Comment (HunseopJeong) [reply src/js/profile/wearable/widget/wearable/SnapListview.js]: If you change to a different page Isn't it common to go to the top of the page?
Could you give me an example of a case you are worried about?

Comment (TomaszLukawskiSam) [reply src/js/profile/wearable/widget/wearable/SnapListview.js]: Hi,
I prepared example for check your PR. (https://github.com/Samsung/TAU/pull/524)
You can test your commit in web browser with slow animations.
image

Comment (HunseopJeong) [reply src/js/profile/wearable/widget/wearable/SnapListview.js]: Thank you to share the example.
I will check it!

**Review by TomaszLukawskiSam — COMMENTED** [review]
Now, works very good. I need to check this PR in several applications.

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Review by singa2000 — APPROVED** [review]
LGTM
I think we need to more check in case of skip to animation of scrollToPosition().

**Pull request merged**